packages/main/src/Select.js

```diff
@@ -456,6 +456,15 @@ class Select extends UI5Element {
 		this._toggleRespPopover();
 	}
 
+	_onclick(event) {
+		this.getFocusDomRef().focus();
```

Comment:
There is  getFocusDomRef() in UI5Element.js #687, you can just set data-sap-focus-ref attr to the "{id}-label" and getFocusDomRef will return it.